packages/core/admin/admin/src/components/AuthenticatedApp/utils/api.js

```diff
@@ -38,6 +38,9 @@ const fetchStrapiLatestRelease = async (toggleNotification) => {
 
 const fetchAppInfo = async () => {
   try {
+    console.warn(
```

Comment:
Should we only print these in development mode for now? Since we haven't started the migration this will print many messages in the console that might frighten our users.